rpa-recorder/6.1/modules/ROOT/pages/documents.adoc

```diff
@@ -31,20 +30,19 @@ The created PDF is an additional documentation of a recording and contains the f
 ** Screenshots: Either screenshots taken on mouse clicks or manual screenshots
 ** Info: Detailed information based upon the action type
 
-The table previously described displays a gray background row whenever a new activity is created, the proceeding rows have a white background to indicate that the sub elements are part of this activity.
+The table previously described displays a gray background row whenever a new activity is created. The proceeding rows have a white background to indicate that the sub elements are part of this activity.
```

Comment:
"proceeding rows" do we mean "preceding rows" (the rows before the gray ones) or something else?